core/math/aabb.h

```diff
@@ -355,12 +355,13 @@ bool AABB::smits_intersect_ray(const Vector3 &p_from, const Vector3 &p_dir, real
 
 void AABB::grow_by(real_t p_amount) {
 
-	position.x -= p_amount;
-	position.y -= p_amount;
-	position.z -= p_amount;
-	size.x += 2.0 * p_amount;
-	size.y += 2.0 * p_amount;
-	size.z += 2.0 * p_amount;
+	position.x -= 0.5 * p_amount;
+	position.y -= 0.5 * p_amount;
+	position.z -= 0.5 * p_amount;
+	size.x += p_amount;
+	size.y += p_amount;
+	size.z += p_amount;
+
```

Comment:
`clang-format` doesn't like this empty line before the closing bracket :)
https://travis-ci.org/godotengine/godot/jobs/417196721

Reply:
Yep, I knew it and I'm using Godot's clang rules in VS2017, but looks like the latest update (VS2017 v15.8) broke something as I'm having problems with others projects...
As usual, I always forget to wait before updating any Micro$oft product...